org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js

```diff
@@ -0,0 +1,187 @@
+/* jshint esversion: 6 */
+/* jshint node: true */
+/* jshint sub:true */
+// This code uses ES2015 syntax that requires at least Node.js v4.
+// For Node.js ES2015 support details, reference http://node.green/
+
+"use strict";
+var request = require('request-promise');
+
+/**
+* This translator class implements the "Hub" interface.
+*/
+class Translator {
```

Comment:
Looks like this translator is not implementing the OCF style getHubResURI and postHubResURI(value) methods? We should since that's a pattern (see the nest and wink thermostat translators for reference)